runner/Dockerfile

```
@@ -25,12 +25,50 @@ FROM build AS binary
+
+RUN home-manager switch
+
+RUN nix shell nixpkgs#gnused --command sh -c '\
+    sed -i '\''s|TZ = "Asia/Tokyo";|TZ = "Pacific/Honolulu";|g'\'' /root/.config/home-manager/home.nix && \
+    sed -i '\''s|LANG = "ja_JP.UTF-8";|LANG = "en_US.UTF-8";|g'\'' /root/.config/home-manager/home.nix && \
+    sed -i '\''s|LC_ALL = "ja_JP.UTF-8";|LC_ALL = "en_US.UTF-8";|g'\'' /root/.config/home-manager/home.nix && \
+    home-manager switch'
+
+RUN nix build --impure --no-link --print-out-paths -f '<nixpkgs>' cowsay pokemonsay lolcat figlet > /tmp/prebuilt-tools-paths.txt
+
+RUN mkdir -p /app
```

Hadolint (2.14.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)